### PR TITLE
smooth out method naming

### LINKED ISSUE #160: Smooth out method naming
There's currently very little consistency in method naming: some use `get_` or `derive_` prefix while some don't, depending on nothing in particular. Let's tidy that up in either direction

### PR DESCRIPTION
**Pull request by garg-tanishka**
fixes #160

### CODE REVIEW

File: git_machete/cmd.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def tupled(f: Callable[..., T]) -> Callable[[Any], T]:
     return lambda tple: f(*tple)
 
 
-def get_second(pair: Tuple[str, str]) -> str:
+def avoid_time_lag(pair: Tuple[str, str]) -> str:
```
Comment (PawelLipski) [inline git_machete/cmd.py:53]: 😯 why specifically `avoid_time_lag`?

**Comment by PawelLipski** [comment]
@garg-tanishka thanks for your contribution! pls check the CI above (`Log In with GitHub` to CircleCI), there are multiple errors from `mypy` type checker... you can run it locally by using `tox`, as outlined in CONTRIBUTING.md :)

File: git_machete/cmd.py [hunk 2]
```diff
@@ -626,7 +626,7 @@ def add(cli_ctxt: CommandLineContext, b: str) -> None:
         expect_in_managed_branches(onto)
 
     if b not in local_branches(cli_ctxt):
-        rb: Optional[str] = get_sole_remote_branch(cli_ctxt, b)
+        rb: Optional[str] = sole_remote_branch(cli_ctxt, b)
```
Comment (BartoszSiwinski) [inline git_machete/cmd.py:629]: sole_remote_branch name suggests that it is a variable containing branch, where in reality it is a method returning such. By removing 'get_' from the names of method it now looks like they these methods are variables. It's established programming convention to name methods as actions (like 'get_string_length' or 'notify_police') or questions (like 'is_even(number) or 'are_commits_in_the_same_format'). I don't recommend introducing this change and similar ones. Applies to all removal of 'get_' or 'is_' from methods and functions names.

Comment (BartoszSiwinski) [reply git_machete/cmd.py:629]: Side Note, python itself is rather into the idea of using 'is' methods names. See https://docs.python.org/3/library/stdtypes.html#float.is_integer or https://docs.python.org/3/library/functions.html#isinstance

**Comment by PawelLipski** [comment]
Also! please apply the changes onto `develop`, as `master` is used for keeping the released changes, and is pretty far behind `develop`

**Comment by garg-tanishka** [comment]
So, should I close this PR?

**Comment by PawelLipski** [comment]
Up to you! it might be more convenient for you to close this one, create a new branch out of `develop` and create a new PR indeed

**Comment by garg-tanishka** [comment]
Yes. Thankyou!

**Pull request closed without merging**